odoo/_monkeypatches/monkeypatches.py

```diff
@@ -72,10 +66,5 @@ def literal_eval(expr):
 
     return orig_literal_eval(expr)
 
-ast.literal_eval = literal_eval
-
-if MIN_PY_VERSION >= (3, 12):
```

Comment:
Moving this will have an impact on runbot, we are parsing this file to find this line.

Reply:
Moved back here.